inst/rmarkdown/templates/wellcomeor/resources/template.tex

```diff
@@ -0,0 +1,143 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Welcome to Overleaf --- just edit your article on the left,
+% and we'll compile it for you on the right. If you give
+% someone the link to this page, they can edit at the same
+% time. See the help menu above for more info. Enjoy!
+%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% For more detailed article preparation guidelines, please see: http://wellcomeopenresearch.org/for-authors/article-guidelines and http://wellcomeopenresearch.org/for-authors/data-guidelines
+
+\documentclass[10pt,a4paper,twocolumn]{article}
+\usepackage{WellcomeOR_styles}
+
+% hyperref required
+\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
+
+% Link coloring
+\hypersetup{$if(colorlinks)$
+            ,
+            colorlinks=true,
+            citecolor=$if(citecolor)$$citecolor$$else$blue$endif$,
+            urlcolor=$if(urlcolor)$$urlcolor$$else$blue$endif$,
+            linkcolor=$if(linkcolor)$$linkcolor$$else$blue$endif$
+            $endif$
+}
+
+
+% Pandoc requirement for specific list requirement
+\providecommand{\tightlist}{%
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+
+%% Default: numerical citations
+\usepackage[numbers]{natbib}
+
+%% Uncomment this lines for superscript citations instead
+% \usepackage[super]{natbib}
+
+%% Uncomment these lines for author-year citations instead
+% \usepackage[round]{natbib}
+% \let\cite\citep
+
+% Pandoc toggle for numbering sections (defaults to be off)
+$if(numbersections)$
+$else$
+\setcounter{secnumdepth}{0}
+$endif$
```

Comment:
Not sure to understand why we need this 🤔 

Can you explain from your tries ? 

Sorry if I already asked, but I don't remember.

Reply:
Because I'm not too familiar with latex and pandoc templates, I just copied some code that seemed to do what I wanted and remove section numbering. I think I've cleaned it up in https://github.com/rstudio/rticles/pull/436/commits/4bb1beb4a38f526a4d0cf64db4949f73bbe50b40 to set that as a default without the unnecessary `if` command

Reply:
Sections numbers are supposed to be off by default ? Wasn't it the case in the default ? 

It would be better to stick with Pandoc's template and set `number_sections` to FALSE in the function, just in case the default change